lib/phoenix/channel/transport.ex

```diff
@@ -214,15 +214,21 @@ defmodule Phoenix.Channel.Transport do
   configured it will return the given `Plug.Conn`. Otherwise a 403 Forbidden
   response will be send and the connection halted.
   """
-  def check_origin(conn) do
+  def check_origin(conn, opts \\ []) do
+    import Plug.Conn
```

Comment:
It's not done all that often, but perfectly acceptable. Imports will only be scoped to the function definition